### ISSUE TITLE
Can I setup colour range in this way in figtree 

### ISSUE THREAD

**Issue by biozhajie** [rambaut/figtree#181]
Dear @rambaut 

Can I setup colour range based on an attribute values (e.g. rates) in this way: from 0 to 1, corresponding to blue to white; from 1 to 10, corresponding to white to red.

Best,
Jie

**Comment by rambaut** [comment]
No, sorry. You could scale the attributes so 1 to 10 gets mapped to 1 to 2 (so the value 1 is the mid-point). You would then edit the scale legend accordingly.

**Comment by biozhajie** [comment]
Dear @rambaut 

Thanks your reply very much! I don't understand the exact meaning of your suggestion, why the 1 to 10 mapped to 1 to 2 will make the value 1 being the the mid-point?

Best,
Jie

**Comment by biozhajie** [comment]
Dear @rambaut 

I have tried to scale the attributes so 0 to 10 gets mapped to 0 to 2 in the branch colour scheme in Apparance panel, but the legend colour scheme doesn't scaled to the new range, how to edit the scale legend accordingly?

Best,
Jie
